cloud/azure/bin/deploy

```diff
@@ -1,56 +1,37 @@
 #! /usr/bin/env bash
 set -e
 
-source "cloud/shared/bin/lib/health.sh"
+source "cloud/shared/bin/lib.sh"
 source "cloud/azure/bin/lib.sh"
 
+# loads and validates config variables and generates tfvars file
 python3 "cloud/shared/bin/lib/tf_apply_setup.py"
-readonly TF_VAR_FILE="setup.auto.tfvars"
-readonly BACKEND_VARS_FILENAME="backend_vars"
-
-cloud/azure/bin/setup_tf_shared_state "${TERRAFORM_TEMPLATE_DIR}/${BACKEND_VARS_FILENAME}"
-
-readonly USER_ID="$(azure::get_current_user_id)"
 
+azure::set_common_vars
```

Comment:
ugggh i realllly wish we could use the config loader class in python for this

Reply:
yeah, that would be preferable huh. do you see a good way to do that?